File: cjoli.Server/Models/Scores.cs
using cjoli.Server.Dtos;

namespace cjoli.Server.Models
{
    public class Scores
    {
        public List<ScoreSquad> ScoreSquads { get; set; } = new List<ScoreSquad>();
        public Dictionary<int, Score> ScoreTeams { get; set; } = new Dictionary<int, Score>();
        public required Score ScoreTourney { get; set; }
        public List<BetScoreDto> BetScores { get; set; } = new List<BetScoreDto>();
    }
}

File: cjoli.client.common/src/models/Scores.ts
import { Bet } from "./Bet";
import { Score } from "./Score";
import { ScoreSquad } from "./ScoreSquad";

export interface Scores {
  scoreSquads: ScoreSquad[];
  scorePhases: Record<number, Score[]>;
  scoreTeams: Record<number, Score>;
  scoreTeamsSeason: Record<number, Score>;
  scoreTeamsAllTime: Record<number, Score>;
  scoreTourney: Score;
  scoreSeason: Score;
  scoreAllTime: Score;
  bet: Bet;
}
